### DISCUSSION TITLE
Is it possible to train Tortoise-tts on a new language?

### DISCUSSION THREAD

**Discussion by Buubiye** [neonbjb/tortoise-tts#516] [Q&A]
Good Morning everyone,

I would like to know is it possible to train Tortoise-tts on a new language, how much data corpus do i need to train tortoise-tts on a new language like Amharic language, also please tell me how much compute power and time it will take to complete this project.

I'm new to Voice cloning so please help me with this.

Thanks,
Buubiye

**Comment by I-Have-No-Idea-What-IAmDoing** [comment]
See here:
https://github.com/neonbjb/tortoise-tts/issues/226#issuecomment-1366927910
https://github.com/neonbjb/tortoise-tts/issues/493
https://github.com/neonbjb/tortoise-tts/issues/442
